src/components/CustomSidebar.jsx

```diff
@@ -0,0 +1,123 @@
+import { Avatar, Box, Typography } from "@mui/material";
+import {
+  ProSidebar,
+  Menu,
+  MenuItem,
+  SubMenu,
+  SidebarContent,
+} from "react-pro-sidebar";
+import Divider from '@mui/material/Divider';
```

Comment:
You can merge, line number 1 and 9, using destructing

```
import { Avatar, Box, Typography, Divider } from "@mui/material";

```

Instead of importing divider in line no 9, add it in first only